.github/workflows/publish-docker.yml

```diff
@@ -8,18 +8,59 @@ on:
         required: true
 
 jobs:
-  tag-docker:
+  docker-moonbeam-release:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
         uses: actions/checkout@v2
         with:
           fetch-depth: 0
+      - name: Download moonbeam binary from GH release
```

Comment:
if it is from release tags, why not simply do a curl ?

Reply:
Yes, it should work with a curl too, but using this action seems to me more readable